changelog.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## Unreleased
+
+### General
+
+-   Ensure GNUInstallDirs cache vars are included before first used (#2778, #2779)
```

Comment:
I thought the changelog only linked to PRs and not issues. I'm surprised to see 2778 in here.

Reply:
Na, when more than one issue number is linked, it's usually the issue and the PR. Maybe you got confused because I was lazy and just used one type of URL on the website page, as I couldn't be bothered to figure out which one was an issue and which one a PR and GitHub does the redirect for us. 😄